### PR TITLE
Update dbcc-traceon-trace-flags-transact-sql.md

### PR DESCRIPTION
**Pull request by brettshearer**
Please research and document this new traceflag - this document should be kept up to date on the same day the trace flag becomes active and published.

### CODE REVIEW

**Comment by PRMerger8** [comment]
@brettshearer : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by ktoliver** [comment]
#label:"aq-pr-triaged"

File: docs/t-sql/database-console-commands/dbcc-traceon-trace-flags-transact-sql.md [hunk 1]
```diff
@@ -165,6 +165,7 @@ The following table lists and describes the trace flags that are available in [!
...
 |<a name="tf8099"></a>**8099**|Enables a spinlock contention fix for high-end systems running [!INCLUDE[sql-server-2019](../../includes/sssql19-md.md)] serving many concurrent users.<br /><br />**Note:** This trace flag applies to [!INCLUDE[sql-server-2019](../../includes/sssql19-md.md)] CU2 and CU3 only. Starting with [!INCLUDE[sql-server-2019](../../includes/sssql19-md.md)] CU4, this behavior is enabled by default. For more information, see this [Microsoft Support article](https://support.microsoft.com/kb/4538688).<br /><br />**Scope**: global only| 
+|<a name="tf8099"></a>**8101**|Fixes 'severe spinlock'https://support.microsoft.com/en-us/topic/kb4538688-fix-severe-spinlock-contention-occurs-in-sql-server-2019-43faea65-fdcb-6835-f7fe-93abdb235837
```
Comment (MikeRayMSFT) [inline docs/t-sql/database-console-commands/dbcc-traceon-trace-flags-transact-sql.md:168]:
```suggestion
|<a name="tf8101"></a>**8101**|Fixes severe spinlock contention in SQL Server 2019 [KB 4538688](https://support.microsoft.com/topic/kb4538688-fix-severe-spinlock-contention-occurs-in-sql-server-2019-43faea65-fdcb-6835-f7fe-93abdb235837).
```

**Comment by MikeRayMSFT** [comment]
@brettshearer  - thank you for the submission. The TF is documented. Working with engineering partners to coordinate timely updates. #please-close

**Pull request closed without merging**